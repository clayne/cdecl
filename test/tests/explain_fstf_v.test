cdecl @ @ -xc++ @ explain friend static void f() @ 65
