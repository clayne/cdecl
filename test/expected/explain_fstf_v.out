explain friend static void f()
                           ^
28: error: "friend static" is illegal for function
